lalrpop/src/normalize/mod.rs

```diff
@@ -19,7 +19,7 @@ macro_rules! return_err {
         return Err(NormError {
             message: format!($($args),+),
             span: $span
-        });
+        })
```

Comment:
Isn't there a compatibility warning about this?

Reply:
The original code makes warning. The changed one is fine.